test/framework/options.py

```diff
@@ -2970,6 +2970,28 @@ def test_stop(self):
         regex = re.compile("COMPLETED: Installation STOPPED successfully", re.M)
         self.assertTrue(regex.search(txt), "Pattern '%s' found in: %s" % (regex.pattern, txt))
 
+    def test_fetch(self):
+        options = EasyBuildOptions(go_args=['--fetch'])
+
+        self.assertTrue(options.options.fetch)
+        self.assertEqual(options.options.stop, 'fetch')
+        self.assertEqual(options.options.modules_tool, None)
+        self.assertTrue(options.options.ignore_osdeps)
+
+        args = ['toy-0.0.eb', '--fetch']
+        stdout, stderr = self._run_mock_eb(args, raise_error=True, strip=True, testing=False)
```

Comment:
@MisterFruits You should add `self.modtool = None` above this line to avoid that the prepared modules tool is passed down (that's also why the test fails)

Reply:
I didn't notice your comment, I took me some time to understand the trick here x)
In 12498c12b509fb645f33b7e7e10fd22f80bb111f I restored back the original ``self.modtool`` after ``self._run_mock_eb(...)``. Not sure it is required, I was afraid I could brake other tests.